dev/ci/presubmits/shared/utils.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# Copyright 2025 The Kubernetes Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+
+
+def get_repo_root():
```

Comment:
could we have a shared/utils common to both dev/ci and dev/tools .. not sure how python imports work across dirs.

Reply:
The typical way to do this would involve a different directory structure with a common root directory that organizes the python modules into reusable packages. The approach in this PR is trying to work within the current directory structure.

You could workaround / hack it by injecting files from other directories into the python module search path, but I'd advise against that

Reply:
also just for the record I'm a big fan of Python but I think a shell language would be a more practical choice for a lot of this tooling

Reply:
Approving this - we can address it in a follow-up PR if needed.